punishments.js

```diff
@@ -1098,6 +1098,8 @@ Punishments.checkLockExpiration = function (userid) {
 	const punishment = Punishments.userids.get(userid);
 
 	if (punishment) {
+		let userObj = Users(userid);
```

Comment:
Why are you calling this variable `userObj` and not `user`.

What would you expect a variable named `user` to contain, if not a user object?